platform/uwp/detect.py

```diff
@@ -138,18 +138,18 @@ def configure(env):
     ## Compile flags
 
     env.Prepend(CPPPATH=['#platform/uwp', '#drivers/windows'])
-    env.Append(CPPFLAGS=['/DUWP_ENABLED', '/DWINDOWS_ENABLED', '/DTYPED_METHOD_BIND'])
-    env.Append(CPPFLAGS=['/DGLES_ENABLED', '/DGL_GLEXT_PROTOTYPES', '/DEGL_EGLEXT_PROTOTYPES', '/DANGLE_ENABLED'])
+    env.Append(CPPDEFINES=['UWP_ENABLED', 'WINDOWS_ENABLED', 'TYPED_METHOD_BIND'])
+    env.Append(CPPDEFINES=['GLES_ENABLED', 'GL_GLEXT_PROTOTYPES', 'EGL_EGLEXT_PROTOTYPES', 'ANGLE_ENABLED'])
```

Comment:
@vnen Are those defines actually used? `'GL_GLEXT_PROTOTYPES', 'EGL_EGLEXT_PROTOTYPES', 'ANGLE_ENABLED'`
Grep doesn't give me any match, but maybe they're used in system headers?

Reply:
I think the prototypes are used in ANGLE code. For `ANGLE_ENABLED` it's probably not used anywhere. It could be used for conditions in the renderer, but we can add again if needed.